codegen/smithy-aws-go-codegen/src/main/resources/software/amazon/smithy/aws/go/codegen/customization/apigw_exports_exceptions/com.amazonaws.amplifybackend#AmplifyBackend.exceptions.json

```diff
@@ -0,0 +1,35 @@
+{
+    "notpicked": [
```

Comment:
This seems like it will be too hard to maintain. Why not just have some top-level filter to say only specific namespaces are impacted?